FindAIETools.cmake

```diff
@@ -147,6 +147,26 @@ foreach(comp ${AIETools_FIND_COMPONENTS})
 		set(aieVersionSpecificPath "unknown")
 	endif()
 
+	if(${comp} STREQUAL "AIE")
+		set(aieVersionTargetPath "target")
+	elseif(${comp} STREQUAL "AIE2")
+		set(aieVersionTargetPath "target_aie_ml")
+	elseif(${comp} STREQUAL "AIE2P")
+		set(aieVersionTargetPath "target_aie2p")
+	else()
+		message(ERROR "${comp} not supported")
+		set(aieVersionTargetPath "unknown")
```

Comment:
Maybe just:
else()
  set(aieVersionTargetPath "target_${comp}")
 
 because the 'special cases' are AIE and AIE2?